# HG changeset patch
# User William Astle <lost@l-w.ca>
# Date 1581026645 25200
#      Thu Feb 06 15:04:05 2020 -0700
# Node ID dd9c5cef2e8032c13b5d45a8468981e3da5b7cb5
# Parent  7e8298f7bc0a300aed8933b8f19486b536ba7b85
Fix missing extern declaration

Variable declarations in headers need to be "extern". Fix missing "extern"
in input.h. (Turns it it was there previously and was erroneously removed in
a previous clean up patch.)

Thank you to John W. Linville <linville@redhat.com> for reporting the
problem.

diff -r 7e8298f7bc0a -r dd9c5cef2e80 lwasm/input.h
--- a/lwasm/input.h	Sun Oct 27 12:06:01 2019 -0600
+++ b/lwasm/input.h	Thu Feb 06 15:04:05 2020 -0700
@@ -46,6 +46,6 @@
 	struct ifl *next;
 };
 
-struct ifl *ifl_head;
+extern struct ifl *ifl_head;
 
 #endif
